freqtrade/optimize/optimize_reports.py

```diff
@@ -21,13 +21,14 @@ def generate_text_table(data: Dict[str, Dict], stake_currency: str, max_open_tra
     tabular_data = []
     headers = [
         'Pair',
-        'Buy Count',
+        'Buys',
```

Comment:
can we call this Trades? 

i think that would fit better - no?

Reply:
Buys are more specific to it and make it easier to understand I think, because there's another table below it that shows Sells and Sell reasons.

Reply:
Sells should probably become 'Closed Trades' or something like this when we'll implement trading shorts 😉

Reply:
mh good point - if we consider longs and shorts, then "buys" is better as we can have "sells / Shorts" too (not now - we don't have that yet).